### DISCUSSION TITLE
Error reporting on the screen

### DISCUSSION THREAD

**Discussion by sany3001** [BitMaker-hub/NerdMiner_v2#327] [Ideas]
If possible, please indicate on screen (maybe on some new error screen):
* wifi connection issue
* internet connection issue
* invalid BTC address
* pool connection issue
* ...

**Comment by valshebnik** [comment]
How can I indicate it?

![photo_2024-02-14_13-00-55](https://github.com/BitMaker-hub/NerdMiner_v2/assets/1078393/69a8d557-01ae-4433-8355-c5f623a4eeee)

I did nothing and the problem resolved itself.

**Reply by sany3001** [reply]
I do not understand your question, sorry. What and where you want to indicate?

**Comment by sany3001** [comment]
Maybe error screen could be optionally activated in case of error (configurable?), by default it would be last screen like "status". So one could see what all is ok and what is not only when interested.
